server/runtime/src/main/server/bin/report.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+PROGNAME=$(basename "$0")
+DIRNAME=$(dirname "$0")
+
+count() { echo $#; }
+
+# Get the PID
+if [ "x$1" = "x" ]; then
+  # There may be more than one
+  PPID=$(pgrep server.sh)
```

Comment:
PPID is read-only

Reply:
Interesting that it doesn't happen here. Maybe some different defaults for when bash is called as sh on different OSs.